src/NeuralNetwork/NeuralNetwork.js

```diff
@@ -251,7 +251,7 @@ class NeuralNetwork {
       const modelJsonResult = await axios.get(filesOrPath.model, {
         responseType: "text",
```

Comment:
The response `.data` is already a string specifically because of the `responseType: "text"` here.  If you don't specify a `responseType` then `axios` will default to parsing it as JSON.